pyproject.toml

```diff
@@ -26,6 +26,8 @@ dependencies = [
   "xarray",
   "flox",
   "scipy",
+  "sparse",
+  "opt-einsum",
```

Comment:
It might be better to make these dependencies optional. Putting them under an "accel" group like in xarray (and document clearly that depending on your computer architecture it could make a difference).

Sparse requires numba and thus llvmlite which aren't the lightest dependencies.